packages/config/config/devices/0x010f/fgpb101.json

```diff
@@ -333,45 +314,65 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "No Action",
+					"label": "No action",
 					"value": 0
 				},
 				{
-					"label": "Switch On",
+					"label": "Switch on",
 					"value": 1
 				},
 				{
-					"label": "Switch Off",
+					"label": "Switch off",
 					"value": 2
 				},
 				{
-					"label": "Toggle Switch On/Off",
+					"label": "Toggle switch on/off",
 					"value": 3
 				}
 			]
 		},
 		"25": {
-			"label": "Key Pressed 3 times – value of SWITCH ON command sent to 3rd association group",
-			"description": "1-99 Set Level; 255 Last State",
+			"label": "Triple Click: Command to Association Group 3",
+			"description": "Allowable range: 1-99",
 			"valueSize": 2,
 			"minValue": 1,
 			"maxValue": 255,
-			"defaultValue": 255,
+			"defaultValue": 99,
 			"unsigned": true,
 			"options": [
 				{
-					"label": "Max Brightness",
-					"value": 99
-				},
-				{
-					"label": "Last State",
+					"label": "Prior State",
 					"value": 255
 				}
 			]
 		},
+		"1[0x04]": {
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Triple Click: Send Scene Events",
+			"defaultValue": 1
+		},
+		"1[0x08]": {
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Four Clicks: Send Scene Events",
+			"defaultValue": 1
+		},
+		"1[0x10]": {
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Five Clicks: Send Scene Events",
+			"defaultValue": 1
+		},
+		"1[0x20]": {
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Key Held: Send Scene Events",
+			"defaultValue": 1
+		},
+		"1[0x40]": {
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Key Released: Send Scene Events",
+			"defaultValue": 1
+		},
```

Comment:
I think we have more specific templates for these in the Fibaro folder - see FGS213 (I think)

Reply:
FGS223. We'd have to override the labels or redfine all of them because they say Key S1 and Key S2 and this only has one button. I'm going to leave it until later.

Reply:
gotcha